### PR TITLE
BN_init for standard integer

### PR DESCRIPTION
**Pull request by kojo1**
Pre-allocated mp_int, mp_digit in BIGNUM

### CODE REVIEW

File: wolfssl/openssl/bn.h [hunk 1]
```diff
@@ -37,18 +37,16 @@
...
-#ifndef MP_MAX_BITS
-    #define MP_MAX_BITS (4098*2)
-#endif
-
 typedef struct WOLFSSL_BIGNUM {
     int neg;        /* openssh deference */
     void *internal; /* our big num */
 #if defined(USE_FAST_MATH) && !defined(HAVE_WOLF_BIGINT)
     fp_int mp;
+#elif defined(WOLFSSL_MT_PREALLOC)
```
Comment (JacobBarthelmeh) [inline wolfssl/openssl/bn.h]: typo on macro *MP

File: wolfcrypt/src/integer.c [hunk 2]
```diff
@@ -421,6 +421,10 @@ int mp_grow (mp_int * a, int size)
   int     i;
   mp_digit *tmp;
 
+#ifdef WOLFSSL_MP_PREALLOC
```
Comment (JacobBarthelmeh) [inline wolfcrypt/src/integer.c]: This puts the restriction on all mp_int's even if not involved with WOLFSSL_BIGNUM. Can this restriction be put at a spot where it only effects WOLFSSL_BIGNUM structures?

Comment (kojo1) [reply wolfcrypt/src/integer.c]: preAlloc flag in mp_int to manage it dynamically, now.

**Comment by JacobBarthelmeh** [comment]
Sent email with findings on review

File: wolfssl/openssl/bn.h (hunk 1 above)
Comment (kojo1) [reply wolfssl/openssl/bn.h]: fixed

**Pull request closed without merging**